### ISSUE TITLE
Add new template variables for multi-disc sets

### ISSUE THREAD

**Issue by eharris** [whipper-team/whipper#569]
It would be nice if there were different disc_template and track_template config for multi-disc sets (where the Total Discs is not equal to 1). Perhaps disc_template_multi and track_template_multi, that by default take the same config as the non-multi templates if they aren't specifically set.

**Comment by eharris** [comment]
Even better, would to be able to provide a different `disc_template` and `track_template` config for multi-disc sets (where the Total Discs is not equal to 1).  Perhaps `disc_template_multi` and `track_template_multi`, that by default take the same config as the non-multi templates if they aren't specifically set.

**Comment by marioqxx** [comment]
I had the same question, but found a solution for me with the use of:
- %M: total number of discs in the chosen release
- %N: number of current disc
in *_template-definition. I use: %%N-%%t %n for the trackname. I.e. the first digit is the CD-number, followed by the track-number (separated with a "-") and finally followed by the track name.

**Comment by eharris** [comment]
Yeah, I found %M and %N, but this is asking to have a set of templates that applies only for multi disc sets, so that I don't have to have a prepended disc number when there is only 1 disc.
